updates/development-updates/december-29th-2023.md

```diff
@@ -0,0 +1,32 @@
+# December 29th 2023
+
+Explore what new changes the Dev team has deployed in the last week!
+
+This can be anything from new features, bug fixes, or QoL changes!
+
+<details>
+
+<summary><strong>New features and items</strong></summary>
+
+* Improved User Settings page with optional email invites
+ 
+</details>
+
+<details>
+
+<summary><strong>Bug fixes and chores</strong></summary>
+
+* Auto genearte org slugs on org creation
+* Fix Get Ticket action for Kaseya BMS
+
+</details>
+
+<details>
+
+<summary><strong>In review, testing and development</strong></summary>
+
+* SentinelOne integration refactor
+* Improved Time Saved functionality with Task level Time Saved and persistence beyond workflow execution retention
+* Bug fixes for App Platform
```

Comment:
Might be worth mentioning the Microsoft improvements vaguely

Reply:
Good suggestion, added.